docs/DeveloperGuide.adoc

```diff
@@ -297,33 +297,32 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new task |
```

Comment:
By removing persons object from your project, does this mean that there's no longer a address book functionality at all?